### ISSUE TITLE
Pane (active) border style changes are ignored

### ISSUE THREAD

**Issue by vdumoulin** [catppuccin/tmux#289]
### Is there an existing issue outlining your problem?

- [X] I have searched the existing issues and they do not solve my problem.

### Describe your problem.

The (active) pane border style keeps its default appearance of gray and orange despite being overridden in the configuration.

### Paste your configuration.

```sh
set -g @plugin 'catppuccin/tmux#latest'
set -g @catppuccin_pane_border_style "fg=#{thm_green}"
set -g @catppuccin_pane_active_border_style "fg=#{thm_blue}"
set -g @plugin 'tmux-plugins/tpm'
```

### What tmux version are you seeing the issue on?

tmux 3.2a

### Any additional comments?

After digging, I believe the root cause is `catppuccin.tmux` is missing the following lines:
```sh
add_tmux_batch_option "@catppuccin_pane_border_style"
add_tmux_batch_option "@catppuccin_pane_active_border_style"
```
The `get_interpolated_tmux_batch_option` function does not see the two options in `tmux_batch_options` and returns the default value no matter what.

**Comment by vdbe** [comment]
Thank you for the issue.

I have 1 question are u using `set -g @plugin 'catppuccin/tmux#latest'`  or `set -g @plugin 'catppuccin/tmux'`?
`set -g @plugin 'catppuccin/tmux#latest'` normally shouldn't have the batching yet.
Changes nothing about the issue just something for future issues and releases I might need to check.

**Comment by vdumoulin** [comment]
Thank you for the quick turnaround!

I was using `set -g @plugin 'catppuccin/tmux#latest'`, but I now realize that I used to use `set -g @plugin 'catppuccin/tmux'` and just recently switched over, but I did not manually remove the repo and reinstall it with `tpm`, so I think I've been at `HEAD` all along.

I just manually removed the repo and reinstalled `catppuccin/tmux#latest`, and I can confirm that the issue does not arise at the latest release.
